src/main/java/net/minecraftforge/java_provisioner/DiscoMain.java

```diff
@@ -74,43 +77,48 @@ private static List<String> l(String...strings) {
     }
 
     private static boolean downloadJdk(
-        int javaVersion, Disco.Arch arch, OS os, Disco.Distro distro,
+        int javaVersion, Disco.Arch arch, OS os, Distro distro,
         boolean auto, File cache
     ) {
-
         if (arch == Disco.Arch.UNKNOWN) {
             arch = Disco.Arch.X64;
-            Log.warn("Unknown Architecture (" + System.getProperty("os.arch") + ")" +
+            System.out.println("Unknown Architecture (" + System.getProperty("os.arch") + ")" +
                 " Defaulting to " + arch.name() + "." +
                 " Use --arch to specify an alternative.");
         }
 
         if (os == OS.UNKNOWN) {
             os = OS.LINUX;
-            Log.warn("Unknown Operating System (" + System.getProperty("os.name") + ")" +
+            System.out.println("Unknown Operating System (" + System.getProperty("os.name") + ")" +
                 " Defaulting to " + os.name() + "." +
                 " Use --os to specify an alternative.");
         }
 
-        Log.info("Downloading JDK:");
-        Log.info("    Version: " + (javaVersion == -1 ? "latest" : javaVersion));
-        Log.info("    Arch:    " + (arch   == null ? "null" : arch  .name()));
-        Log.info("    OS:      " + (os     == null ? "null" : os    .name()));
-        Log.info("    Distro:  " + (distro == null ? "null" : distro.name()));
-        Log.info("    Cache:   " + cache.getAbsolutePath());
+        System.out.println("Downloading JDK:");
+        System.out.println("    Version: " + (javaVersion == -1 ? "latest" : javaVersion));
+        System.out.println("    Arch:    " + (arch   == null ? "null" : arch  .name()));
```

Comment:
This all needs to still run through a loggin system of some kinda even if its just spitting it out to System.out at the end of the day. Because we need to be able to invoke this from other processes via command line. Or atleast add the --log <file> option so it gets saved to file.

Reply:
I would rather add a `--log` option. From Gradle and our tools, we have the ability to intercept the standard output, so I did not think it would be that big of a deal.